src/controllers/AuthController.js

```diff
@@ -0,0 +1,90 @@
+import bcrypt from 'bcrypt'
+import jwt from 'jsonwebtoken'
+import dotenv from 'dotenv'
+import {User} from '../models/UserModel.js'
+
+
+dotenv.config()
+
+const AuthController = {
+    signup: async (req, res) => {
+        const {firstName, lastName, email, password, confirmPassword} = req.body 
+
+        try {
```

Comment:
Codacy has a fix for the issue: [Replace `········` with `····`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
    try {
```